setup.py

```diff
@@ -278,6 +280,33 @@ def build_skia(build_base):
     return build_dir
 
 
+def get_skia_using_pkgconfig():
+    """Runs `pkg-config --libs --cflags skia` and parses returned
+    flags using argparse.
+    """
+    _parser = argparse.ArgumentParser()
+    _parser.add_argument("-I", dest="include_dirs", action="append", default=[])
+    _parser.add_argument("-L", dest="library_dirs", action="append", default=[])
+    _parser.add_argument("-l", dest="libraries", action="append", default=[])
+
+    pkgconfig = os.environ.get("PKG_CONFIG", "pkg-config")
+    log.info("Finding skia using pkg-config")
+    try:
+        op = subprocess.run(
+            [pkgconfig, "--cflags", "--libs", "skia"],
+            text=True,
+            check=False,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        stout = op.stdout
+    except FileNotFoundError:
+        stout = ""
+    args, _ = _parser.parse_known_args(shlex.split(stout))
+    return args
+
+
+pkg_config_op = get_skia_using_pkgconfig()
 if BUILD_SKIA_FROM_SOURCE:
```

Comment:
when BUILD_SKIA_FROM_SOURCE we do not want to check pkg-config at all, we want to build a standalone libskia from source as part of the build and embed that in the wheel. This should be the default.

Reply:
I have done it in recent changes https://github.com/fonttools/skia-pathops/pull/54/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R292.